messages/es.json

```diff
@@ -226,15 +226,15 @@
   },
   "screens.ServerStatus.timeoutDesc": {
     "description": "Description when the server has not responded for 10 seconds",
-    "message": "Algo pasa con el base de datos Mapeo"
+    "message": "Si sigues viendo este mensaje, quizá tengas que reiniciar Mapeo"
   },
   "screens.ServerStatus.timeoutTitle": {
     "description": "Title of message when the server has not responded for 10 seconds",
-    "message": "Esperando contacto con el base de datos de Mapeo, si continúa este mensaje, reinicie a Mapeo"
+    "message": "Algo está pasando con la base de datos Mapeo"
```

Comment:
@gmaclennan @aliya-ryan @jencastrodoesstuff This change will alter the meaning of this translation, do we want users to know that they should try to restart mapeo or was this on purpose?

Reply:
This is correcting a mix between `screens.ServerStatus.timeoutTitle` and `screens.ServerStatus.timeoutDesc` - the description should have the same information that was previously in this title.